modules/tinymce/src/plugins/legacyoutput/test/ts/browser/LegacyOutputPluginTest.ts

```diff
@@ -1,152 +1,177 @@
-      LegacyUnit.equal(editor.getContent(), '<p>text</p>');
-    });
-
-    suite.test('TestCase-TINY-4741: LegacyOutput: Convert bold to span if styling attributes are present on format removal', (editor) => {
-      editor.setContent('<p><b class="abc" style="color: red; font-size: 20px;" data-test="2">text</b></p>');
-      LegacyUnit.setSelection(editor, 'b', 0, 'b', 4);
-      editor.execCommand('bold');
-      LegacyUnit.equal(editor.getContent(), '<p><span class="abc" style="color: red; font-size: 20px;">text</span></p>');
-    });
-
-    suite.test('TestCase-TBA: LegacyOutput: Formats registered before loading initial content', () => {
-      const formats = formatsCell.get();
-      LegacyUnit.equal(formats.bold[0], { inline: 'b', remove: 'all', deep: true, split: true, preserve_attributes: [ 'class', 'style' ] });
-      LegacyUnit.equal(formats.italic[0], { inline: 'i', remove: 'all', deep: true, split: true, preserve_attributes: [ 'class', 'style' ] });
-      LegacyUnit.equal(formats.underline[0], { inline: 'u', remove: 'all', deep: true, split: true, preserve_attributes: [ 'class', 'style' ] });
-      LegacyUnit.equal(formats.fontname[0], { inline: 'font', toggle: false, attributes: { face: '%value' }, deep: true, split: true });
-    });
-
-    TinyLoader.setupLight((editor, onSuccess, onFailure) => {
-      Pipeline.async({}, Log.steps('TBA', 'LegacyOutput: Test legacy formatting', suite.toSteps(editor)), onSuccess, onFailure);
-    }, {
-      plugins: 'legacyoutput',
-      indent: false,
-      base_url: '/project/tinymce/js/tinymce',
-      font_formats: 'Arial=arial,helvetica,sans-serif;',
-      setup: (editor) => {
-        // Store the formats on `PostRender`, which is fired before the initial editor content is loaded
-        editor.on('PostRender', () => {
-          formatsCell.set({ ...editor.formatter.get() });
-        });
-      }
-    }, success, failure);
-  }
-);
+describe('browser.tinymce.plugins.legacyoutput.LegacyOutputPluginTest', () => {
+  const formatsCell = Cell<any>({});
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'legacyoutput',
+    indent: false,
+    base_url: '/project/tinymce/js/tinymce',
+    font_formats: 'Arial=arial,helvetica,sans-serif;',
+    setup: (editor) => {
+      // Store the formats on `PostRender`, which is fired before the initial editor content is loaded
+      editor.on('PostRender', () => {
+        formatsCell.set({ ...editor.formatter.get() });
+      });
+    }
+  }, [ Plugin, Theme ]);
+
+  it('TBA: Setting overrides', () => {
+    const editor = hook.editor();
+    assert.isFalse(editor.getParam('inline_styles'));
+    assert.equal(editor.getParam('fontsize_formats'), '8pt=1 10pt=2 12pt=3 14pt=4 18pt=5 24pt=6 36pt=7');
+    assert.equal(editor.getParam('font_formats'), 'Arial=arial,helvetica,sans-serif;');
+  });
+
+  it('TBA: Font color', () => {
+    const editor = hook.editor();
+    editor.focus();
```

Comment:
I missed this earlier sorry, but this probably shouldn't be here and instead, it should be set as the last argument in the `TinyHooks.bddSetupLight` call.
```suggestion
```

Reply:
done